docs/DeveloperGuide.md

```diff
@@ -551,12 +629,19 @@ Guarantees: The applicants with name matching the search will be listed.
     * 1a1. Staff-Snap shows an error message.
 
       Use case ends.
+
 * 1b. No applicant found.
```

Comment:
Not too sure if this part should be from step 1 or step 2, since step 1 is just inputting the command, while step 2 is where the applicants get shown 🤔 what do you think about it?